roles/telemetry_verify_metrics/tasks/verify_ceilometer_compute_metrics.yml

```diff
@@ -62,9 +62,9 @@
       until: result.rc == 0 and "ceilometer_cpu" in result.stdout
       changed_when: false
 
-    #todo: Create Test Id in Polarian
     - name: |
         TEST Use openstack observabilityclient to verify ceilometer_power_state compute metrics are stored in prometheus
+        RHOSO-14297
```

Comment:
With the custom_xml plugin, we don't need to have the polarion ID in the task name.
Polarion history indicates that this is already reporting as expected.

Please remove the comment, but don't add the test id